scripts/deploy.sh

```
@@ -0,0 +1,77 @@
+rsync -avz --delete \
+    --exclude='.git' \
+    --exclude='.devenv' \
+    --exclude='.env' \
+    --exclude='*.log' \
+    --exclude='dist/' \
+    --exclude='node_modules/' \
+    --exclude='.github/' \
+    ./ $USER@$SERVER:$DEPLOY_PATH/
+
+echo "2. Building and deploying on server..."
+ssh $USER@$SERVER << EOF
```

Shellcheck (0.11.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)